### ISSUE TITLE
Class-Balanced Loss based on Effectivene number of examples

### ISSUE THREAD

**Issue by ashutosh1919** [keras-team/tf-keras#538]
Currently, Keras has Focal loss which can be one of the options to handle the class imbalance issues. But I myself have faced problems in the case of some datasets like RecSys. IMO, Keras should have Class-balanced loss ([paper](https://arxiv.org/pdf/1901.05555.pdf))

I am interested in working on this issue. If you think we should add this, then please assign the issue to me and I can raise PR.

cc @fchollet

**Comment by sachinprasadhs** [comment]
Thank you for reporting the issue. 
We're now prioritizing [Keras 3](https://keras.io/keras_3/) development, a multi-backend framework that offers significant advantages by supporting `JAX`, `TensorFlow`, `PyTorch` and `OpenVINO`. 
This means we won't be adding new features to `TF-Keras (Keras 2)`. 

`TensorFlow` users can benefit directly from `Keras 3`, which is included by default starting with version `2.16.1`.
 We encourage you to migrate to `Keras 3` and experience its enhanced capabilities. If you feel this specific feature is crucial for the `Keras` framework itself, please open a new issue at: https://github.com/keras-team/keras/issues

For more details and usage guide about Keras 3, refer: https://keras.io/getting_started/about/
